dist/openshift/graph-data-sync.yaml

```
@@ -0,0 +1,64 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: cincinnati
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: cincinnati-graph-data-sync
+      name: cincinnati-graph-data-sync
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: cincinnati-graph-data-sync
+      strategy:
+        type: Recreate
+      template:
+        metadata:
+          labels:
+            app: cincinnati-graph-data-sync
+          name: cincinnati-graph-data-sync
+        spec:
+          containers:
+            - args:
+                - -c
+                - |
+                  set -euo pipefail
+
+                  pip install --user PyYAML
+
+                  cd /tmp
+                  while sleep 600
+                  do
+                    rm -rf blocked-edges channels
+                    curl -L https://github.com/openshift/cincinnati-graph-data/archive/master.tar.gz |
+                      tar -xzv --strip-components=1
```

Since we're only cleaning up "blocked-edges" and "channels" previously, I suggest to filter the extraction by these two as well.

---

We need the `hack` script too.  There's not much in the tarball besides that.

---

I'd rather explicitly whitelist the things we need so let's just add the script to it as well.

---

This should be as simple as

```suggestion
                      tar -xzv --strip-components=1 blocked-edges channels hack/graph-util.py
```